### PR TITLE
Add RSKIP434

### PR DESCRIPTION
**Pull request by julia-zack**

### CODE REVIEW

File: IPs/RSKIP434.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+This mechanism is crucial for maintaining a trust-minimized operation, ensuring that users can independently verify the locking of funds on Bitcoin, which is essential for the security and functionality of cross-chain transactions between Bitcoin and Rootstock.
+
+Chain work in Bitcoin refers to the cumulative computational effort that has been expended to build the blockchain. It is a measure of the total work done by miners to create and validate blocks, ensuring the security and integrity of the blockchain. The chain work is currently stored in a signed 12 Bytes element in the Block store.
+
+As of block #849,138 chainWork has reached a value of 800028897aa1dce163ead2cf which exceeds the capacity of the 12 Bytes storage format.
```
Comment (nathanieliov) [inline IPs/RSKIP434.md:31]:
```suggestion
As of block #849,138, chainWork has reached a value of `0x800028897aa1dce163ead2cf`, which exceeds the capacity of the 12-byte storage format.
```

**Review by nathanieliov — APPROVED** [review]
LGTM! Just a small suggestion.

File: IPs/RSKIP434.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
+rskip: 434
+title: Bridge Bitcoin block chainwork up to 12 Bytes
+created: 26-Jun-24
+author: JD,JZ,MI
+purpose: Sec
+layer: Core
+complexity: 1
+status: Draft
+description: Change in Bridge Bitcoin storage to accept blocks with chainwork serialized up to 12 Bytes
```
Comment (julia-zack) [inline IPs/RSKIP434.md:10]: just a suggestion: i would say `12 _unsigned_ bytes` to be more specific

**Review by apancorb — APPROVED** [review]
LGTM

File: IPs/RSKIP434.md (hunk 1 above)
Comment (julia-zack) [reply IPs/RSKIP434.md:31]: agree with natha,
also #849_138 instead of #849,138

**Review by julia-zack — COMMENTED** [review]
lgtm! (cannot approve since the pr is mine lol)
just a suggestion

**Review by marcos-iov — APPROVED** [review]

File: IPs/RSKIP434.md (hunk 2 above)
Comment (nathanieliov) [reply IPs/RSKIP434.md:10]: Agree

**Pull request merged**